cunumeric/config.py

```diff
@@ -252,3 +253,23 @@ class CuNumericTunable(IntEnum):
     NUM_PROCS = _cunumeric.CUNUMERIC_TUNABLE_NUM_PROCS
     MAX_EAGER_VOLUME = _cunumeric.CUNUMERIC_TUNABLE_MAX_EAGER_VOLUME
     HAS_NUMAMEM = _cunumeric.CUNUMERIC_TUNABLE_HAS_NUMAMEM
+
+
+# Match these to BitGeneratorOperation in bitgenerator_util.h
+@unique
+class BitGeneratorOperation(IntEnum):
+    CREATE = 1
```

Comment:
Please turn this into something like `CREATE = _cunumeric.CUNUMERIC_BITGENOP_CREATE` and define an enum in `cunumeric_c.h` that has `CUNUMERIC_BITGENOP_CREATE` (the same thing applies to other codes.) And you redefine an enum class in the C++ code that mirrors that C enum and uses it in the C++ tasks. This is how we make sure both sides of the code use the same set of codes without needing to renumber the enums for every addition of a field.